src/App.js

```diff
@@ -1,12 +1,70 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import PokemonCard from "./PokemonCard";
+import axios from "axios";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokenames, setPokenames] = useState(null);
```

Comment:
Als dit altijd een array is, dan zou ik ook een lege array als default waarde gebruiken i.p.v. `null`